styles.css

```diff
@@ -0,0 +1,586 @@
+
+  .summit-logo {
+    position: relative;
+    left: 80px;
+  }
+
+  .nav-links {
+    position: relative;
+    left: 465px;
+  }
+
+  .border-link {
+    border: 5px solid #ec5242;
+    padding: 8px 8px;
+  }
+
+  article .headline {
+    display: none;
+  }
+
+  article .headlinedesktop {
+    display: block;
+  }
+
+  .cover-program-desktop {
+    background-image: url('./images/patternbg.png');
+    background-repeat: repeat;
+    background-position: center;
+    background-color: rgb(39, 42, 50);
+    padding: 5px;
+    width: 100%;
+    height: 450px;
+  }
+
+  #main-section .cover-program {
+    display: none;
+  }
+
+  #main-section .cover-program-desktop {
+    display: block;
+  }
+
+  .program-list-desktop h2 {
+    color: #fff;
+    margin-top: 20px;
+  }
+
+  .desktop-wrapper {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .bg-program-desktop {
+    display: flex;
+    flex-direction: column;
+    list-style-type: none;
+    overflow: hidden;
+    margin: 0 5px;
+    padding: 0 10px;
+    height: 250px;
+    margin-top: 10px;
+    background: rgba(255, 255, 255, 0.1);
+    width: 16%;
+  }
+
+  .bg-program-desktop li h3 {
+    color: #ec5242;
+    font-size: 16px;
+    padding-left: 2px;
+    margin-left: 2px;
+    width: auto;
+  }
+
+  .bg-program-desktop li p {
+    font-size: 14px;
+    color: #fff;
+    margin-left: 2px;
+    padding: 2px;
+  }
+
+  .whole-link {
+    color: #fff;
+    text-decoration: underline;
+    margin-top: 30px;
+  }
+
+  #featured-mobile {
+    display: none;
+  }
+
+  #featured-desktop {
+    display: block;
+  }
+
+  .partner-section {
+    height: 300px;
+  }
+}
+
+/* ========Desktop version CSS Rule set=======1 */
+.topdesktop {
+  background-image: url('./images/topbg.png');
+  background-size: cover;
+  height: 200px;
+}
+
+.topheadingdesktop {
+  color: orangered;
+  font-size: 34px;
+  font-weight: 400;
+  position: relative;
+  top: 150px;
+  left: 105px;
+}
+
+.topheadingdesktopdiv {
+  background-image: url('./images/bgimagedesign.png');
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: 350px 0;
+  background-color: #fff;
+  position: relative;
+  left: 65px;
+  padding: 45px;
+  width: 100%;
+  height: 545px;
+}
+
+.topheadingh1desktop {
+  font-size: 50px;
+  display: block;
+  color: red;
+  font-weight: 900;
+  background-image: linear-gradient(to bottom right, red, yellow);
+  background-size: 100%;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: #ec0e0e4f;
+}
+
+.paradivdesktop {
+  background-color: #d3d3d3;
+  border: 5px solid darkgrey;
+  padding: 10px;
+  width: 50%;
+}
+
+.headlineparadesktop {
+  color: #272a31;
+}
+
+.datedivdesktop {
+  width: 100%;
+}
+
+.dateheadingdesktop {
+  color: #272a31;
+  font-weight: 800;
+}
+
+.locationparadesktop {
+  color: #272a31;
+}
+
+table {
+  width: 80%;
+  position: relative;
+  left: 110px;
+}
+
+td {
+  padding: 15px;
+}
```

Comment:
- [ ] KIndly make sure to align the content of the speaker correctly in order to match as much as possible as the provided design.

   - ### YOUR DESIGN
   
    ![image](https://user-images.githubusercontent.com/87197412/219401927-f0c0b4fb-1bf7-4af5-9673-522aaa83fcfa.png)

   - ### PROVIDED DESIGN
    
    ![image](https://user-images.githubusercontent.com/87197412/219401581-9d2c5da9-0367-45af-8fa9-d7de171f7cd1.png)